plugins/modules/site_workflow_manager.py

```diff
@@ -610,8 +630,29 @@ def site_exists(self):
         site_exists = False
         current_site = {}
 
-        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.5.3") <= 0:
+        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.7.6") >= 0:
+            sites = None
+            response = self.get_site(site_name_hierarchy)
+            self.log("Raw response from get_site: {}".format(response), "DEBUG")
+            if isinstance(response, dict):
+                sites = response.get("response", [])
+            elif isinstance(response, list):
+                self.log("Unexpected list returned from get_site, skipping: {}".format(response), "ERROR")
+            else:
+                self.log("Unexpected response type: {}".format(type(response)), "ERROR")
+
+            if not sites:
+                self.log("No site information found for name: {0}".format(response), "WARNING")
+
+            for site in sites:
+                if isinstance(site, dict):
+                    self.log("No site information found for name: {0}".format(self.pprint(site)), "INFO")
```

Comment:
Didn't get the log message.. why do we say "no site information found" ?

Reply:
Modified the log